src/components/shared/FormDialog.tsx

```diff
@@ -24,19 +30,26 @@ export default function FormDialog<T extends FieldValues>({
   discardText,
   confirmText,
   onCloseDialog,
+  handleBack,
   schema,
   onSubmit,
   renderForm
 }: FormDialogProps<T>) {
+  const isSmallScreen = useIsSmallScreen()
+  const { t: translate } = useTranslation()
   const { control, handleSubmit, formState, reset } = useForm<T>({
-    resolver: yupResolver(schema)
+    resolver: schema ? yupResolver(schema) : undefined
   })
+  const steps = useNewDeliverySteps()
+  const { currentStep, isCompletedMove } = useNewDeliveryContext()
```

Comment:
dont use the delivery context inside the `FormDialog` and like we talked we need to get the stepper out of here.

Reply:
After our chat - i don't do anything